erroranalysis/erroranalysis/_internal/matrix_filter.py

```diff
@@ -540,3 +540,9 @@ def fill_matrix_nulls(matrix, null_value):
     idx_tuples = list(zip(idx_arrays[0], idx_arrays[1]))
     for tuple in idx_tuples:
         matrix.iloc[tuple] = null_value
+
+
+def get_py_value(value):
+    if isinstance(value, np.generic):
```

Comment:
Could it be anything other than `np.generic`? Just wondering since numpy has many types. I'm sure you've considered that (hence more of a question), but the reason why it's only `np.generic` isn't obvious to me from this PR.

Reply:
this was the best I could come up with based on these searches:

https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.generic

https://stackoverflow.com/questions/12569452/how-to-identify-numpy-types-in-python#:~:text=To%20get%20the%20type%2C%20use%20the%20builtin%20type,%27numpy%27%20in%20str%20%28type%20%28a%29%29%20Out%20%3A%20True

https://stackoverflow.com/questions/25432523/how-to-check-if-a-value-is-of-a-numpy-type

Reply:
Could you add a docstring explaining why this is needed? It's obvious now, but it may look a little odd in three months time (the `Value '0' is not JSON-serializable` thing has wasted my time in the past as well)

Reply:
This is indeed very interesting and thanks for sharing these links! I agree with @riedgar-ms. Documenting this may turn out to be useful later on.